README.md

```diff
@@ -62,22 +59,29 @@ By default, readabilty is not enforced. To enforce readability:
     *.scala    @abcxyz/java-readability
     ```
 
-1.  Ensure you have [branch protection rules][branch-protection-rules] to
+1. Ensure you have [branch protection rules][branch-protection-rules] to
     require a minimum number of reviewers. Specifically "Require review from
     Code Owners".
 
-
 ## Adding a new language
 
-To add a new language for readability requirements:
+There are two types of languages used by `abcxyz`. Languages with broad adoption that are a standard part of our technology stack (i.e. Go, Terraform, etc.) and languages with limited use or that are in early adoption (i.e. Python, Dart, etc.).
+
+### Limited use / early adoption languages
+
+For new languages that we are uncertain about their widespread adoption, we will add targeted individuals as the CODEOWNERS for new repos. We will only convert them to a group and establish the standard readability practice after the usage of that language exceeds X where X is some combination of number of repositories and lines of code. Currently, X is 5+ repos or 50k+ LoC.
```

Comment:
Should we make the 5+ repos and 50k+ more obvious? I also dont think we need to mention X here as its defined.

Reply:
Pushed an updated version - do you like that better?

Reply:
lgtm